scss/_variables.scss

```diff
@@ -1432,8 +1432,34 @@ $pagination-icon:                   var(--#{$boosted-prefix}chevron-icon) !defau
 $pagination-icon-size:              subtract($spacer * 2, $border-width * 2) !default;
 $pagination-icon-width:             add(.5rem, 1px) !default;
 $pagination-icon-height:            subtract(1rem, 1px) !default;
+
 $pagination-active-item-bg:         $primary !default;
+$pagination-active-item-color:      color-contrast($pagination-active-item-bg) !default;
 $pagination-active-item-border:     $pagination-active-item-bg !default;
+
+// Dark variant
+$pagination-dark-bg:                        $black !default;
```

Comment:
In Bootstrap, CSS dark vars are separated in the docs (https://getbootstrap.com/docs/5.2/components/dropdowns/#variables). I find it interesting, we probably want to do the same.

Reply:
Actually it seems that they don't have a specific method on this part regarding carousel, offcanvas, ... But I agree, we should propose it on their side !